<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="misurazione">
       <attribute name="title">
        <string>Misurazione</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="cavename"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectcave">
            <property name="text">
             <string>Seleziona grotta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newcave">
            <property name="text">
             <string>Nuova grotta</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Distanza</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="distance">
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Direzione</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="heading">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pendenza</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="frontalInclination">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Incl. lato</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sideTilt">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Sinistra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="leftW"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Destra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rightW"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Sopra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="upW"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Sotto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="downW"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="manualMode">
            <property name="text">
             <string>Modalit√† manuale</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Dal punto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromP"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Al punto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toP"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGraphicsView" name="section"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="save">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 255, 0);</string>
              </property>
              <property name="text">
               <string>Salva misurazione</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="pressure"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Temperatura</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="temperature"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Pressione</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabella">
       <attribute name="title">
        <string>Tabella</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="deleterow">
            <property name="text">
             <string>Elimina selezionato</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savetable">
            <property name="text">
             <string>Salva modifiche</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="fulltable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="risultato">
       <attribute name="title">
        <string>Risultato</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="updatedrawing">
            <property name="text">
             <string>Aggiorna</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QGraphicsView" name="pianta"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Pianta</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Spaccato</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGraphicsView" name="spaccato"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="zoom">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Impostazioni</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Salvataggio </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Cartella dei rilievi: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="outputfolder"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QCheckBox" name="openlastcave">
               <property name="text">
                <string>Inizia sempre dall'ultima grotta</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="dxfmesh">
               <property name="text">
                <string>Mesh in DXF</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Data e ora </string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QDateTimeEdit" name="dateTimeEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="savedatetime">
             <property name="text">
              <string>Imposta</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Lidar </string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Utilizza il Lidar per misure laterali</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Esegui media su 45¬∞ per le distanze</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Calibrazione </string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="calibraBussola">
             <property name="text">
              <string>Bussola</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calibraTilt">
             <property name="text">
              <string>Inclinometro</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calibraLaser">
             <property name="text">
              <string>Distanzimetro</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modalit√†</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="exit">
          <property name="text">
           <string>Chiudi il programma</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="accesspoint">
          <property name="text">
           <string>Access point WiFi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shutdown">
          <property name="text">
           <string>Spegni</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reboot">
          <property name="text">
           <string>Riavvia</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Aiuto</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="aiuto">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Charlotte √® un dispositivo sviluppato dal Gruppo Speleologico Talpe del Carso.
Permette di realizzare rilievi topografici vettoriali delle cavit√† naturali e artificiali in modo automatico.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
